Automattic/vip
# feature(export): Add ability to generate a new backup when exporting SQL

## Description

This PR adds the ability to trigger a new database backup 

## Steps to Test

1. Check out PR.
1. Run `rm -rf dist && npm link && node ./dist/bin/vip export sql @7300.production --generate-backup`
